apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  </head>
+  <body>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <header class="presentation">
+      <h1 class="recipe-name">Apple Pies</h1>
+    </header>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+    <main>
+      <section class="description">
+        <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+        <img class="info" src="images/recipe-info.png" alt="recipe-info">
+        <hr>
```

Comment:
Try no to use ```hr``` for styling, only when it makes sense to have a line break from structure for example poetry :). The concept is separation of concerns, css should be responsable to create that space and not html (in charge of strcuture)